openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -400,6 +400,14 @@ export default function init(){
             $('.wmd-preview').before('<h3 id="prevHead" style="margin:15px 0 10px;padding:0;">Preview</h3>');
         }
     });
+    initReadingListFeature(debounce);
+    initBorrowAndReadLinks();
+}
+
+/**
+ * @param {Function} debounce function
+ */
+export function initReadingListFeature(debounce) {
```

Comment:
Why does this take a debounce function as a parameter?

Reply:
Because $('.dropclick').on('click', debounce(function(){ uses a debounce function.
I'm trying to use dependency injection to make it clear how everything fits together -this also supports better unit testing.

Reply:
`debounce` might be better placed in a utils file and then imported. I agree picking it up by closure is awkward, but passing a dependency as a parameter is also awkward :P It actually makes testing a little worse, because the unit test now needs to find an entirely unrelated debounce function to test this function.

Reply:
> because the unit test now needs to find an entirely unrelated debounce function to test this function.
that's exactly what I needed to do to test this - inject a spy debounce function. This makes testing easier. In the cller of initReadingListFeature $.debounce is passed in.

Reply:
That feels hacky now that I think about it. What if debounce is used in another place inside this method later? Then the test will fail (or not fail when it should). Oh well, resolving since I guess this is merged now. I'll fix it myself later.